testing/README.md

```diff
@@ -0,0 +1,75 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
+For further details about how to configure Vault check [the Vault documentation](https://github.com/elastic/infra/tree/master/docs/vault)
+
+1. Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+2. Create a cluster for e2e tests:
+
+In the `testing` directory:
+
+```
+GITHUB_TOKEN=... SLACK_CHANNEL=... make create-cluster
```

Comment:
You need a reference to how to generate the Github token here as well.

Reply:
Yes, please.  For example, it would be good to know whether we should be using the new fine-grained tokens or the classic tokens.  I _think_ it needs to be the latter because I'm not sure the former supports SSO.  In the case of classic tokens, it would be good to know which scopes (if any) must be selected.

Reply:
In step 1 above ("Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)", we specify a slack channel via `oblt-cli configure --slack-channel`.  Do we need to specify it again in this step via the `SLACK_CHANNEL` environment variable?

Reply:
When I try this step, I'm getting a 404 error from the GitHub API.   However, I'm also getting messages in Slack from `oblt-robot` about my cluster(s) being created. Is this the expected behavior for this step?  My GitHub token has SSO enabled for the `elastic` org but no scopes selected (see my comment about that above).

```
$ GITHUB_TOKEN=ghp_XXXXXXXXXXX SLACK_CHANNEL=@XXXXXXXXXX make create-cluster
jq: error: Could not open file cluster-info.json: No such file or directory
oblt-cli cluster create custom --template-file /Users/shaunak/development/github/elastic-agent/testing/test-cluster.yml.tpl --output-file /Users/shaunak/development/github/elastic-agent/testing/cluster-info.json --parameters '{"StackVersion": "8.6.1", "ClusterName": "e2e-elasic-agent-1680795110", "SlackChannel": "@U0J8KCYRY"}' --wait 15
2023/04/06 08:31:50 Using config file: /Users/shaunak/.oblt-cli/config.yaml
2023/04/06 08:31:51 Already up to date.
+-------+--------------------------------------------------------------------------------+
| LEVEL |                                    MESSAGE                                     |
+-------+--------------------------------------------------------------------------------+
| INFO  | Profiling is deployed on release-oblt, dev-oblt, edge-oblt, and edge-lite-oblt |
|       |                                                                                |
+-------+--------------------------------------------------------------------------------+
2023/04/06 08:31:52 SlackChannel: '@U0J8KCYRY'
2023/04/06 08:31:52 User: 'ycombinator'
2023/04/06 08:31:52 Git mode: 'ssh'
2023/04/06 08:31:56 Reading template : /Users/shaunak/development/github/elastic-agent/testing/test-cluster.yml.tpl
2023/04/06 08:31:56 Using template : /Users/shaunak/development/github/elastic-agent/testing/test-cluster.yml.tpl
2023/04/06 08:31:56 Parsing parameters : {"StackVersion": "8.6.1", "ClusterName": "e2e-elasic-agent-1680795110", "SlackChannel": "@U0J8KCYRY"}
2023/04/06 08:31:56 Writing cluster config file : /Users/shaunak/.oblt-cli/observability-test-environments/environments/users/ycombinator/elastic-agent-e2e-pnnzk-custom.yml
2023/04/06 08:31:56 Template parameters: map[ClusterConfigFile:/Users/shaunak/.oblt-cli/observability-test-environments/environments/users/ycombinator/elastic-agent-e2e-pnnzk-custom.yml ClusterName:elastic-agent-e2e-pnnzk-custom Date:2023-04-06T08:31:56-0700 GitOps:false Seed:pnnzk SlackChannel:@U0J8KCYRY StackVersion:8.6.1 TemplateFile:/Users/shaunak/development/github/elastic-agent/testing/test-cluster.yml.tpl User:ycombinator]
2023/04/06 08:31:56
2023/04/06 08:31:56 [main 27e33538] oblt-cli(ycombinator): Create elastic-agent-e2e-pnnzk-custom custom cluster using elastic-agent-e2e template
 1 file changed, 39 insertions(+)
 create mode 100644 environments/users/ycombinator/elastic-agent-e2e-pnnzk-custom.yml
2023/04/06 08:31:58 remote:
remote: Create a pull request for 'oblt-cli/27e33538e09ac7a430e7414ddc0a14c404fd670a' on GitHub by visiting:
remote:      https://github.com/elastic/observability-test-environments/pull/new/oblt-cli/27e33538e09ac7a430e7414ddc0a14c404fd670a
remote:
To github.com:elastic/observability-test-environments.git
 * [new branch]        oblt-cli/27e33538e09ac7a430e7414ddc0a14c404fd670a -> oblt-cli/27e33538e09ac7a430e7414ddc0a14c404fd670a
Error: GET https://api.github.com/repos/elastic/observability-test-environments/pulls: 404 Not Found []
make: *** [create-cluster] Error 1
```

Reply:
Added detailed docs about the `GITHUB_TOKEN` to ingest-dev